priv/repo/migrations/20220503061007_add_unique_email_index.exs

```diff
@@ -0,0 +1,7 @@
+defmodule GoogleSearchDataViewer.Repo.Migrations.AddUniqueEmailIndex do
+  use Ecto.Migration
+
+  def change do
+    create unique_index(:users, [:email])
```

Comment:
We also can include this index in the above migration `GoogleSearchDataViewer.Repo.Migrations`

Reply:
Do you mean include this in the initial User creation Migration file? If so then yes, I only came to realise upon testing that I needed the index on the underlying DB for the unique_constraint changeset functions to work.

Please let me know if you would like me to modify the initial migration / creation from the User